lib/cocoapods/installer/analyzer/sandbox_analyzer.rb

```diff
@@ -103,7 +103,7 @@ def pod_state(pod)
         #
         def pod_added?(pod)
           return true if resolved_pods.include?(pod) && !sandbox_pods.include?(pod)
-          return true if !folder_exist?(pod) && !sandbox.local?(pod)
+          return true if !sandbox.local?(pod) && !folder_exist?(pod)
```

Comment:
`folder_exists?` is a slower path and `sandbox.local?` is fast - short circuiting earlier avoids extra work

Reply:
nice